chia/consensus/block_creation.py

```diff
@@ -216,11 +211,11 @@ def create_foliage(
 
         # Addition Merkle set contains puzzlehash and hash of all coins with that puzzlehash
         for puzzle, coin_ids in puzzlehash_coin_map.items():
-            addition_merkle_set.add_already_hashed(puzzle)
-            addition_merkle_set.add_already_hashed(hash_coin_ids(coin_ids))
+            additions_names.append(puzzle)
```

Comment:
the puzzle_hash is not an addition_name, but it is part of the addition merkle set

Reply:
yeah. what's a better name for this list? `additions_merkle_items`?

Reply:
yeah that sounds good